eng/dependabot/Packages.props

```
@@ -1,30 +1,80 @@
 <Project>
-  <!-- Packages in this file have versions updated periodically by Dependabot. Versions managed by Darc/Maestro should be in ..\Package.props. -->
+
+  <!-- Packages in this file have versions updated periodically by Dependabot. Versions managed by Darc/Maestro should be in ..\Package.props.
+
+  Packages must be set to their package version property if it exists (ex. BenchmarkDotNetVersion) since source-build uses
+  these properties to override package versions if necessary. -->
+
   <ItemGroup>
     <PackageReference Update="System.Diagnostics.Process" Version="4.3.0" />
+    <PackageReference Update="System.Diagnostics.Process" Condition="'$(SystemDiagnosticsProcessVersion)' != ''" Version="$(SystemDiagnosticsProcessVersion)" />
+
     <PackageReference Update="System.IO.Compression" Version="4.3.0" />
+    <PackageReference Update="System.IO.Compression" Condition="'$(SystemIOCompressionVersion)' != ''" Version="$(SystemIOCompressionVersion)" />
+
     <PackageReference Update="System.Runtime.Loader" Version="4.3.0" />
+    <PackageReference Update="System.Runtime.Loader" Condition="'$(SystemRuntimeLoader)' != ''" Version="$(SystemRuntimeLoader)" />
```

```suggestion
    <PackageReference Update="System.Runtime.Loader" Condition="'$(SystemRuntimeLoaderVersion)' != ''" Version="$(SystemRuntimeLoaderVersion)" />
```

Should it be similar in all cases below?